build.gradle

```diff
@@ -113,6 +114,19 @@ sonarqube {
   }
 }
 
+pmd {
```

Comment:
Is this instead of checkstyle?
i.e. this repo already has checkstyle on it

Reply:
Rather a complementary check to be used within Sonar or Codacy.

Reply:
What benefit does it give? How much extra time does it take?

Reply:
It helps finding bad practices and takes as much as caring for quality does. ;)

Reply:
shouldn't sonar be used for this?